assignment5/assignment5.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
+
+function fileRead(file) {
+  return new Promise(function (resolve, reject) {
+    fs.readFile(file, 'utf8', function (err, data) {
+      if (err) {
+        console.log(err);
```

Comment:
2 things here.
- If there is an error, the steps after this will execute. Do printing something to console only stop the executions of this callback?
- Why do we define `reject`? Will you statement `console.log(err)` send the error back .catch of Promise?